mon-pix/app/app.js

```diff
@@ -7,10 +7,15 @@ import loadInitializers from 'ember-load-initializers';
 import Resolver from 'ember-resolver';
 import config from 'mon-pix/config/environment';
 
+import { inspectorSupport } from './inspector-support';
+
 if (config.sentry.enabled) {
   initSentry();
 }
 
+if (config.emberInspector.enabled) {
+  inspectorSupport();
+}
```

Comment:
```suggestion
// This is a temporary solution, see https://github.com/emberjs/ember-inspector/issues/2612
if (config.emberInspector.enabled) {
  inspectorSupport();
}
```

Comme dans orga, rajouter le commentaire informatif ?